spec/factories/annotation_texts.rb

```diff
@@ -6,4 +6,10 @@
     creator { FactoryBot.create(:admin) }
     association :annotation_category, factory: :annotation_category
   end
+
+  factory :annotation_text_with_deduction, parent: :annotation_text do
+    after(:create) do |annotation_text|
```

Comment:
Since you're just setting an attribute here you don't need an `after(:create)`. You can just do `deduction { 1.0 }` as with the other attributes (see above factory).